### PR TITLE
Update intro-integrations.md

### PR DESCRIPTION
**Pull request by jerry-li-dev**
Added information on static IP address support.

### CODE REVIEW

File: src/pages/docs/integrations/intro-integrations.md [hunk 1]
```diff
@@ -35,3 +35,13 @@ You can access integrations by navigating to your workspace in the [web dashbo
...
+
+## Static IP Support for Integrations
+
+You can use static IP addresses to enable integrations that need to access hosted (private) networks behind firewalls that require preapproved IP addresses.
+
+Postman supports implementing static IP addresses for the following integration services:
+
+- [Custom Webhooks](https://learning.postman.com/docs/integrations/webhooks/)
+- [GitHub Custom Domain Backup](https://learning.postman.com/docs/integrations/available-integrations/github/#backup-collections-to-github-on-custom-domain)
+- [GitLab Custom Domain Backup](https://learning.postman.com/docs/integrations/available-integrations/gitlab/#backup-your-postman-collections-to-gitlab-on-a-custom-domain)
```
Comment (shashankawasthi88) [inline src/pages/docs/integrations/intro-integrations.md:47]: Can we have general instructions here too? As it is common across all the Integrations which support static IPs.

File: src/pages/docs/integrations/intro-integrations.md [hunk 2]
```diff
@@ -35,3 +35,13 @@ You can access integrations by navigating to your workspace in the [web dashbo
 ![Workspace Integrations](https://assets.postman.com/postman-docs/workspace-integrations-api.jpg)
 
 Select __Browse Integrations__. Click __View Details__ to see an overview of an integration then add it to your workspace.
+
+## Static IP Support for Integrations
+
+You can use static IP addresses to enable integrations that need to access hosted (private) networks behind firewalls that require whitelisted IP addresses. You will need to whitelist the static IP `3.212.102.200` in your firewall to enable data backup integrations. All Postman network calls to your domain will come from this IP address. Whitelisting this IP address will allow the integrations work as expected.
```
Comment (shashankawasthi88) [inline src/pages/docs/integrations/intro-integrations.md:41]: Same comment for All Postman network calls.

File: src/pages/docs/integrations/intro-integrations.md [hunk 3]
```diff
@@ -35,3 +35,21 @@ You can access integrations by navigating to your workspace in the [web dashbo
...
+
+You can use static IP addresses to enable integrations that need to access hosted (private) networks behind firewalls that require whitelisted IP addresses.
+
+If your network is behind a firewall that requires whitelisted IP addresses, you will need to use a static IP address to enable collection backups to integrations on custom domains.
+
+Contact your IT team to whitelist the following static IP in your firewall to enable collection backup integrations:
+
+US East: `3.212.102.200`
+
+All calls for each integration will come from this IP address. Whitelisting this IP address will allow the integrations work as expected.
```
Comment (claireriley) [inline src/pages/docs/integrations/intro-integrations.md:49]: "allow the integrations work" -> "allow the integrations to work"

File: src/pages/docs/integrations/intro-integrations.md [hunk 4]
```diff
@@ -35,3 +35,21 @@ You can access integrations by navigating to your workspace in the [web dashbo
...
+
+## Static IP Support for Integrations
+
+You can use static IP addresses to enable integrations that need to access hosted (private) networks behind firewalls that require whitelisted IP addresses.
+
+If your network is behind a firewall that requires whitelisted IP addresses, you will need to use a static IP address to enable collection backups to integrations on custom domains.
+
+Contact your IT team to whitelist the following static IP in your firewall to enable collection backup integrations:
+
+US East: `3.212.102.200`
```
Comment (claireriley) [inline src/pages/docs/integrations/intro-integrations.md:47]: I might put this in a bullet point just so it's indented

File: src/pages/docs/integrations/intro-integrations.md [hunk 5]
```diff
@@ -35,3 +35,21 @@ You can access integrations by navigating to your workspace in the [web dashbo
...
+
+All calls for each integration will come from this IP address. Whitelisting this IP address will allow the integrations work as expected.
+
+Postman supports implementing static IP addresses for the following integration services:
```
Comment (claireriley) [inline src/pages/docs/integrations/intro-integrations.md:51]: "integration services" I'd consider just "integrations"

File: src/pages/docs/integrations/intro-integrations.md (hunk 3 above)
Comment (claireriley) [inline src/pages/docs/integrations/intro-integrations.md:49]: I might clarify the first sentence—I assume in order for this to happen they need to select this option when setting up the integration? I would briefly mention that's a condition e.g. "Once you ..., all calls ..."

**Review by claireriley — CHANGES_REQUESTED** [review]
Looks good—few small notes! 😄

File: src/pages/docs/integrations/intro-integrations.md [hunk 6]
```diff
@@ -35,3 +35,19 @@ You can access integrations by navigating to your workspace in the [web dashbo
 ![Workspace Integrations](https://assets.postman.com/postman-docs/workspace-integrations-api.jpg)
 
 Select __Browse Integrations__. Click __View Details__ to see an overview of an integration then add it to your workspace.
+
+## Static IP Support
+
+You can use static IP addresses to enable integrations and custom webhooks for Postman collections backups that need to access hosted (private) networks behind firewalls that require whitelisted IP addresses.
```
Comment (claireriley) [inline src/pages/docs/integrations/intro-integrations.md:41]: Two tiny things—

- I think the rule is if "collections" is preceded by "Postman", we want to capitalize the C
- I think we want to remove the s on "collections"

**Review by claireriley — APPROVED** [review]
Two letters to adjust! Looks good so I'm approving, you can make that small change and then merge to develop!

**Pull request merged**